utils/threshAlgos.py

```diff
@@ -1508,14 +1508,13 @@ def getDetName(entry):
                     if type(graph) == r.TH1F :
                         graph = r.TGraph(graph)
 
-                    # make sure the yaxis range is restricted
                     graph.GetYaxis().SetRangeUser(1e-1,1e8)
 
                     # Draw a line on the graphs
                     kneeLine.append(r.TLine(dict_dacInflectPts[dacName][ohKey][vfat][0], 1e-1, dict_dacInflectPts[dacName][ohKey][vfat][0], 1e8) )
                     kneeLine[vfat].SetLineColor(2)
                     kneeLine[vfat].SetVertical()
-                    canv_Summary1D.cd(chamber_vfatPos2PadIdx[gemType][vfat] )   
+                    canv_Summary1D.cd(chamber_vfatPos2PadIdx[gemType][vfat] )
```

Comment:
still extraneous whitespace inside the function call (actually, also on L1514, now that I notice)

Reply:
Sorry, my style versus repo style. I fixed it.